src/main/scala/com/actian/spark_vector/colbuffer/timestamp/TimestampColumnBuffer.scala

```diff
@@ -86,15 +86,16 @@ private class TimestampTZConverter extends TimestampConversion.TimestampConverte
   private final val TimeMaskBI = new BigInteger(timeMask)
   private final val SecondsInMinuteBI = BigInteger.valueOf(SecondsInMinute)
 
-  /** Set the 117 most significant bits to 1 and the 11 least significant bits to 0. */
+  /** Set the 117 most significant bits to 1 and the 11 least significant bits to 0.
+   *  @note the mask has to be in BigEndian due to its use on BigIntegers */
   private def timeMask: Array[Byte] = {
```

Comment:
I don't really see the use of this anymore. In convert we shift by 11 bytes and so the and(TimemaskBI) would at most lose some of the most significant bits (which I think would be a bug to do so). I think we can remove it.